backend/models/bcf.js

```diff
@@ -397,9 +397,24 @@ async function getIssueBCF(issue, account, model, unit) {
 								X: vp.up[0],
 								Y: -vp.up[2],
 								Z: vp.up[1]
-							},
-							FieldOfView: vp.fov * 180 / Math.PI
+							}
 						};
+
+						if ("orthogonal" === vp.type && vp.orthographicSize) {
+							viewpointXmlObj.VisualizationInfo.OrthogonalCamera = {
+								...camera,
+								ViewToWorldScale: vp.orthographicSize
+							};
+						}
+
+						if (("perspective" === vp.type || !_.get(vp, "extras._noPerspective")) && vp.fov) {
+							viewpointXmlObj.VisualizationInfo.PerspectiveCamera = {
+								...camera,
+								FieldOfView: vp.fov * 180 / Math.PI
+							};
+						}
+					} else if (_.get(vp, "extras.OrthogonalCamera")) {
+						viewpointXmlObj.VisualizationInfo.OrthogonalCamera = _.get(vp, "extras.OrthogonalCamera");
```

Comment:
what kind of things would be stored here?

Reply:
@carmenfan This is for legacy support in case anybody imported a BCF with an orthogonal camera in the past